### ISSUE TITLE
Airline inactive colors unreadable?

### ISSUE THREAD

**Issue by bigpick** [pineapplegiant/spaceduck#43]
# Info
Heya! Been looking for something to get me away from my main themes (Dracula, snazzy, nord) from time to time, and I really think this colorscheme's palette looks great, while still being functional/able to be easily consumed esp wrt syntax (my main issue with nord). I'm hoping I can contribute some more ports/themes in my spare time, and really hope this theme gets the hype it deserves.

**Issue**: By default, the inactive colors as part of shipped colors of the vim-airline theme that come with the spaceduck repo are really hard to read: https://github.com/pineapplegiant/spaceduck/blob/0d06e20f8390b58de3e69a1ac5c43d2ca833ce39/autoload/airline/themes/spaceduck.vim#L32-L34

Using `#1b1c36` as the foreground against a `#16172d` background makes the text have almost no contrast:

![Screen Shot 2021-06-04 at 9 37 51 AM](https://user-images.githubusercontent.com/9803299/120809978-9e42d880-c518-11eb-93c1-270ff9b42fb3.png)

I don't think changing the background would be the best move, so changing the foreground color seems to be the best option. Of the ones I tried, I think `#30365F` looked best:

```vim
let s:inactive1 = [ "#30365F", "#16172d", 234, 234 ]
let s:inactive2 = [ "#30365F", "#16172d", 234, 234 ]
let s:inactive3 = [ "#30365F", "#16172d", 234, 234 ]
```
![Screen Shot 2021-06-04 at 9 39 08 AM](https://user-images.githubusercontent.com/9803299/120810113-c2061e80-c518-11eb-9e67-17d283d5092f.png)

---
... though, that said, I'm not sure I think the other colors of the airline bar look the best either, but that is likely just personal preference (and it expecting to match one of the screenshots of the other status bars at least in closeness, but does not)

1. how come the normal background for the bar is Dark Purple2 (`#686f9a`, which is one of the lighter colors) while most of the other status bars seem to be dark background colored (including the ones in the main screenshot of the repo)?
2.  Using the Dark Purple2 as bg for the main parts, the foreground (which is just dark purple `#30365F`) is also really hard to read imo. If sticking with the dark purple2 background, cursor line `#16172d` or bg `#0f111b` seem like better choices imo. Though, I think the better option would being changing the background to be dark, and using lighter foreground coloring to stick with the feel of the theme overall more.
3. why use `#000000` for foreground text where provided instead of just the background coloring, `#0f111b`?
4. I would also suggest once finalized, to get the theme added to the [vim-airline-themes](https://github.com/vim-airline/vim-airline-themes) repo, too, for extra exposure. I think shipping the airline theme with the repo itself makes sense if the repo is specifically for vim, but if the intent is to have much more ports/etc available, I don't think it'd make sense to keep the contents in here. But I guess this gets more into the organizational/stylistic approach of how the theme is maintained, which probably is a bigger thing to talk about then just this specific part.

**Comment by pineapplegiant** [comment]
You bring up a lot of good points that I do wanna address, but I wanna respond quickly. 

Mmm @jinh0 has been working on the airline theme. Can you check out the dev branch and let me know what you think? We’re working on expanding the color palette and we fixed some issues with readability (like on comments) on that branch that I’m hoping to push to main this weekend.

**Comment by bigpick** [comment]
Awesome, thanks, I'll take a look at that.

For the organizational stuff, have you thought much about if (or how if so) to go towards that route, or no? I guess, just would be curious if in the meantime developing ports for other things, what the best way to then "get them added to the theme" would be best (currently, looks like just links to others personal repo's in the main README)

**Comment by bigpick** [comment]
The dev branch airline looks *a lot better* wrt the other bullets I pointed out (also like the warning color being toned down much more too), but [the issue with the inactive text is still there](https://github.com/pineapplegiant/spaceduck/compare/dev#diff-9fa153d009cbacc50dcdde44396768196ec57e21fea7fc99ce3eda762d05be3eR32-R34):

![Screen Shot 2021-06-04 at 11 03 58 AM](https://user-images.githubusercontent.com/9803299/120822829-cf290a80-c524-11eb-944f-bcf452c99db4.png)

**Comment by pineapplegiant** [comment]
I’ve thought about it a little bit (I didn’t expect people to like the theme and contribute at all 😅). 

I really like nord and how they handle things, such a well designed theme (although the colors are too muted imo). So I’ve thought about creating a gatsby site like how they have done and in the mean time moving everything non plugin related to the wiki. 

I’m open to any suggestions, of course.

**Comment by jinh0** [comment]
Hey @bigpick , thanks for taking the time to use the dev branch! Glad you're liking my airline. Thanks for letting me know about the inactive text being too dark. I'll fix it this weekend! 😊

**Comment by bigpick** [comment]
I agree, nord's site is pretty nice. I think that though also includes a lot of additional work to maintain a good looking site (kinda true for any approach though so), and from what I can tell, two places of information for a given port? 

e.g nord-vim has its own repo under the org, but on the site (https://www.nordtheme.com/ports/vim) it has its own web page that isn't tied at all wrt source/infromation/etc to the nord-vim repo itself, so when one is updated, the other also then needs to get updated as well (nord-docs)

I don't know that it is a necessarily large burden, and may actually be viewed as a benefit, as keeping the two separate helps avoid needing to include submodule type things, which tends to be love/hate.

Coming from mostly using/looking at Dracula+contributing there sometimes, I guess the other sort of option for the organizational stuff would be like how Dracula does it, which similar to nord is a repo with repos for each port and a main repo for the "theme itself"/website, but then how things are managed are just slightly different. That said, I think Dracula's site is probably more simplistic (not a dig) than Nord's, but I think I like how they manage the individual ports better (I seem to get the feeling that most of the nord stuff has to go through the articicestudio entity, whereas the Dracula things, you don't see Zeno required to be personally approving/merging everything). To be fair, I've not really looked at Nord's much, so all of those opinions may be totally wrong though, so maybe I am just ignorant on most things Nord still

**Comment by jinh0** [comment]
Hey @bigpick I used your fix for the inactive colors and also replaced the #000000 with #0f111b like you mentioned. Thanks!

Regarding the organization of the ports, I think that is a bigger discussion that should be its own issue or post on the discussions section. If I may put in my two cents however, I think a big Dracula-like website is a far stretch as of now since Spaceduck is still figuring out the colors to use.
